### ISSUE TITLE
Написать статью про полезные плагины и плюсы IDE (например, про VSCode?)

### ISSUE THREAD

**Issue by MedvedevNik** [Yandex-Practicum/extra-content#4]
Предлагаю закинуть студентам эту инфу.
Т.к. есть такое ощущение, что студенты не особо разобрались с Emmet'ом даже. 
Рассказать как сократить время написания верстки (типа ul.class>li.class-item*3...)

Также о пользе использовании различных плагинов, типа претиера и т.д

**Comment by pa7lux** [comment]
идея замечательная. возможно отделил бы эммет от тулзов всяких, но да! в этом большой кайф

**Comment by pa7lux** [comment]
сейчас в программе есть про это вебинар, возможно часть этой работы уже проделана.

Призову-ка я @dannileb посмотреть эту issue, может его заинтересует

**Comment by dannileb** [comment]
Да, в первом вебинаре мы как раз говорим про это: тулзы, плагины и прочее. Даже настраиваем VSC вместе со студентами.  Мне кажется, кто заинтересуется, пойдёт изучать и кастомизировать после веба как ему нужно. А кто нет, дай бог на вебе всё сделает

Может, интереснее будет рассказать про IDEшки разные, например почему VSC а не WebStorm или InteliJ какая-нибудь
